Automattic/jetpack
# Charts: Adding a theme provider

Fixes https://github.com/Automattic/jetpack-roadmap/issues/2041

## Proposed changes:

* extending Automattic Charts with a theme provider, allowing for easier chart adoption
* replacing hard-coded colours in the chart components

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* install Jetpack packages if needed (this PR doesn't add new packages) `pnpm i`
* navigate to the directory and run storybook: `cd projects/js-packages/charts` and `pnpm run storybook`
* verify that existing charts are working and new theme provider is available
* verify that different themes load different colours

![SCR-20241211-mwct](https://github.com/user-attachments/assets/b84cdad9-804c-4f3c-96de-8382920f1392)
